datasets/allen-cell-imaging-collections.yaml

```diff
@@ -28,3 +29,12 @@ Resources:
     ARN: arn:aws:s3:::quilt-aics
     Region: us-east-1
     Type: S3 Bucket
+DataAtWork:
+  - Title: Allen Cell Feature Explorer
+    URL: https://www.allencell.org/cell-feature-explorer.html
+    AuthorName: Multiple (see link)
```

Comment:
I would honestly make this "Allen Institute for Cell Science"

Everyone contributed to making this data available. :)